packages/cli/src/index.ts

```diff
@@ -85,10 +85,10 @@ function printHelpInformation(
 }
 
 function printUnknownCommand(cmdName: string) {
-  const suggestion = didYouMean(
-    cmdName,
-    commander.commands.map(cmd => cmd._name),
-  );
+  const availableCommands = commander.commands.map(cmd => cmd._name);
+  const suggestion = availableCommands.find(cmd => {
+    return leven(cmd, cmdName) < 3;
```

Comment:
I don't have any preference here whether `3` is a good number or not. `didyoumean` has [threshold of 40% by default](https://github.com/dcporter/didyoumean.js#threshold), so in order to not change the behaviour, we would need to use a different number.

Reply:
I guess replacing this with `< cmd * 0.4` would be equivalent. It's a good UX question.

Reply:
That would result in `NaN`. FYI `jests-cli` makes use of the same threshold: https://github.com/facebook/jest/blob/5d1e98beda19dcfff3f10557396098524efbb639/packages/jest-validate/src/utils.ts#L55

Reply:
The intention here was `cmd.length * 0.4` and it makes sense. I'm gonna merge that, because I don't care too much. But feel free to improve this further :)